coverage/resources.csv

```diff
@@ -1,164 +1,4 @@
-,account_registry,transfer_ownership,0,223,0.0,0.0,"(1,)",{},0.0
-,account_registry,transfer_ownership,0,223,0.0,0.0,"(1070856479481634900007870680840527701197761158322858021165525254969064253486,)",{},0.0
-,account_registry,transfer_ownership,0,223,0.0,0.0,"(1,)",{},
-,account_registry,transfer_ownership,0,223,0.0,0.0,"(1070856479481634900007870680840527701197761158322858021165525254969064253486,)",{},0.0
-,account_registry,transfer_ownership,0,223,0.0,0.0,"(1070856479481634900007870680840527701197761158322858021165525254969064253486,)",{},0.0
-,account_registry,transfer_ownership,0,223,0.0,0.0,"(1,)",{},
-,account_registry,transfer_ownership,0,223,0.0,0.0,"(1,)",{},0.0
-,account_registry,transfer_ownership,0,223,0.0,0.0,"(1070856479481634900007870680840527701197761158322858021165525254969064253486,)",{},0.0
-,account_registry,transfer_ownership,0,223,0.0,0.0,"(1,)",{},0.0
-,account_registry,transfer_ownership,0,223,0.0,0.0,"(1070856479481634900007870680840527701197761158322858021165525254969064253486,)",{},0.0
-,account_registry,transfer_ownership,0,223,0.0,0.0,"(1070856479481634900007870680840527701197761158322858021165525254969064253486,)",{},
-,account_registry,transfer_ownership,0,223,0.0,0.0,"(1,)",{},
-,account_registry,transfer_ownership,0,223,0.0,0.0,"(1070856479481634900007870680840527701197761158322858021165525254969064253486,)",{},
-,account_registry,transfer_ownership,0,223,0.0,0.0,"(1,)",{},0.0
-,account_registry,set_account_entry,21,190,6.0,0.0,"(7735, 9171)",{},2.0
-,account_registry,get_evm_contract_address,10,98,3.0,0.0,"(7735,)",{},1.0
-,account_registry,get_starknet_contract_address,11,96,3.0,0.0,"(9171,)",{},1.0
-,kakarot,set_account_registry,0,93,0.0,0.0,(),{'registry_address_': 2886833052963579273547603311562511163286573173333730118165592388204472041034},
-,kakarot,set_account_registry,0,93,0.0,0.0,(),{'registry_address_': 2886833052963579273547603311562511163286573173333730118165592388204472041034},
-,kakarot,set_account_registry,0,93,0.0,0.0,(),{'registry_address_': 2886833052963579273547603311562511163286573173333730118165592388204472041034},
-,kakarot,set_account_registry,0,93,0.0,0.0,(),{'registry_address_': 2886833052963579273547603311562511163286573173333730118165592388204472041034},0.0
-,kakarot,set_account_registry,0,93,0.0,0.0,(),{'registry_address_': 2886833052963579273547603311562511163286573173333730118165592388204472041034},0.0
-,kakarot,set_account_registry,0,93,0.0,0.0,(),{'registry_address_': 2886833052963579273547603311562511163286573173333730118165592388204472041034},0.0
-,kakarot,set_account_registry,0,93,0.0,0.0,(),{'registry_address_': 2886833052963579273547603311562511163286573173333730118165592388204472041034},0.0
-,kakarot,set_account_registry,0,93,0.0,0.0,(),{'registry_address_': 2886833052963579273547603311562511163286573173333730118165592388204472041034},0.0
+context,contract_name,function_name,n_memory_holes,n_steps,args,kwargs
```

Comment:
seems like you didn't run the tests locally, did you?

Reply:
I was able to run on old tests format, I share the results on telegram, but could share here

Reply:
still, if you ran `make test` locally, you should update this file with full data. you ran a partial test locally I guess